src/manifold/src/quickhull.h

```diff
@@ -0,0 +1,826 @@
+#ifndef QUICKHULL_HPP_
+#define QUICKHULL_HPP_
```

Comment:
I think you should add the copyright header from the upstream library (or make one for them), as well as change the define to `pragma once` that we typically use.

Reply:
Should I add both Manfiold copyright header and the upstream header? (also they did mention that we don't need to add anything so what should I add)

Reply:
I think we can include both.